tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py

```diff
@@ -612,7 +612,6 @@ def read(self, data_id):
     def read_item(self, data_id):
         return DataRepresentation(*self.read_dispatcher(data_id), identifier=data_id)
 
```

Comment:
please return empty line (recommended for making code more readable, usually for classes used 2 line spacing)

Reply:
Fixed